example/ios/Podfile

```
@@ -2,10 +2,9 @@ source 'https://cdn.cocoapods.org/'
 source 'https://github.com/m0nac0/flutter-maplibre-podspecs.git'
 
 # Uncomment this line to define a global platform for your project
-platform :ios, '11.0'
+platform :ios, '12.0'
```

```suggestion
platform :ios, '11.0'
```

This bump is not needed I think.

---

True, we are now using the version 11.